### ISSUE TITLE
[Accessibility] Source Tree should have a more efficient keyboard navigation.

### ISSUE THREAD

**Issue by yzen** [devtools-html/debugger.html#4068]
Places that need improvement:

- [ ] Keyboard focus should be kept on the tree container. This way if the user presses tab/shif+tab the focus moves to a next/previos widget outside the list. This helps with skipping sources if the user is not interested in stepping inside the tree.
- [ ] There's right now a keyboard trap when the user tabs to the last tree item. There is no way to tab forward or backwards any more.
- [ ] In tree navigation should just be performed by arrow keys exclusively. Up/down to navigate between tree items. Right/left to expand/collapse the tree item.
- [ ] Currently selected source item should be tracked using aria-activedescendant [attribute][a]

[a]: https://www.w3.org/WAI/GL/wiki/Using_aria-activedescendant_to_allow_changes_in_focus_within_widgets_to_be_communicated_to_Assistive_Technology

### Screenshot

![Screen Shot 2017-09-18 at 8.09.39 PM.png](https://shipusercontent.com/54338689712fedefd518f4334cc38941/Screen%20Shot%202017-09-18%20at%208.09.39%20PM.png)

**Comment by yzen** [comment]
The same solution as #4067 should do the trick. Updating to the latest devtools-core tree component should fix all keyboard issues.

**Comment by jasonLaster** [comment]
Great, I or @nchevobbe will do the update soon.

**Comment by jasonLaster** [comment]
we updated!
